src/uwloc/data/wav.py

```diff
@@ -1,30 +1,54 @@
 import logging
-from datetime import datetime
+from datetime import datetime, timezone
 from typing import Tuple
 
 import numpy as np
 import numpy.typing as npt
+import pytz
 import scipy.io.wavfile as wf
 import wavinfo
 
 logger = logging.getLogger(__name__)
 
 
+def parse_date(dstr: str) -> datetime:
+    assert "(UTC" in dstr, "Expected a timezone suffix of the form: (UTC-<offset>)"
+    DATE_FORMAT = "%H:%M:%S %d/%m/%Y"
+    parts = dstr.split(" ")  # TIME, DATE, TIMEZONE
+    sdate_notz = " ".join(parts[0:2])
+    offset_hr = int(parts[2].replace("(UTC", "").replace(")", "").strip() or 0)
+    date = (
+        datetime.strptime(sdate_notz, DATE_FORMAT)
+        .replace(tzinfo=pytz.FixedOffset(offset_hr * 60))
+        .astimezone(timezone.utc)
+    )
+    return date
```

Comment:
I think this can really be improved and cleaned up using the `dateutil` package as we discussed in Slack like so:

```python
from dateutil import parser
from datetime import datetime
import pytz

def parse_date(dstr: str) -> datetime:
    # Positive due to unix artifact
    # See https://pubs.opengroup.org/onlinepubs/7908799/xsh/tzset.html
    # See https://stackoverflow.com/questions/11984618/why-does-python-return-a-negative-timezone-value
    dt = parser.parse(dstr, fuzzy=True)
    date = (dt + dt.tzinfo.utcoffset(dt)).replace(tzinfo=pytz.utc)
    return date
```

Reply:
@lsetiawan will this work correctly in windows too?

Reply:
Not 100% sure. Might have to check and see! I'm assuming so since it is using the underlying Python library which works with things underneath as "unix" based time. From Python doc:

```
time.timezone

    The offset of the local (non-DST) timezone, in seconds west of UTC (negative in most of Western Europe, positive in the US, zero in the UK). See note below.

```

Reply:
Just checked this. I can confirm it works on windows! 😄